doc/examples/filters/plot_deconvolution.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+=====================
+Image Deconvolution
+=====================
+In this example, we deconvolve an image using Richardson-Lucy
+deconvolution algorithm.
+
+The algorithm is based on Bayesian-based iterative method
+where an image blurred by a point-spread function is 
+recovered. 
+
+.. [1] William Hadley Richardson, "Bayesian-Based Iterative 
+       Method of Image Restoration",
+       J. Opt. Soc. Am. A 27, 1593-1607 (1972), doi:10.1364
+
+.. [2] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import color, data, restoration
+
+from scipy import ndimage as ndi
+
+astro = color.rgb2gray(data.astronaut())
+psf = np.ones((5, 5)) / 25
+astro_blurred = ndi.convolve(astro, psf)
+astro += 0.1 * astro.std() * np.random.standard_normal(astro.shape)
+
+deconvolved = restoration.richardson_lucy(astro, psf, 5)
+
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(8, 5), sharex=True, sharey=True)
+
+ax1.imshow(astro)
```

Comment:
In my environment all imshow calls plot in non-grayscale colormap by default.
You should add `, cmap=plt.cm.gray)` or any other workaround.